core/server/master/src/main/java/alluxio/master/file/loadmanager/LoadJob.java

```diff
@@ -66,26 +68,40 @@ public class LoadJob {
   private final Map<String, Status> mFailedFiles = new HashMap<>();
 
   private int mBandwidth;
+  private boolean mVerificationEnabled;
 
   private LoadStatus mStatus;
-  private List<FileInfo> mFiles;
+  private Optional<List<FileInfo>> mFiles = Optional.empty();
```

Comment:
nit: Is it possible to use empty list rather than Optional.empty for the default value?  I guess you just want to use Optional.empty to represent "not started" or "no result"

Reply:
Optional.empty() represents not started. Technically it's possible that the results of listing is an empty list.